### PR TITLE
Updates merging rule to be less strict on merging from original author

### PR DESCRIPTION
**Pull request by b-m-f**
As discussed in the dev catchup it should be possible to merge a PR when everything looks good, even if you are not the original author.

Branch cleanup will be done in the Fork.

### CODE REVIEW

File: workflow/github.md [hunk 1]
```diff
@@ -82,7 +82,7 @@ Here are the steps that need to be completed before merging:
 - Your pull request must be approved by at least one person
 - Any automated checks must have passed
 - The original author should merge a pull request
-  - This can be overruled when necessary, only if the author is unavailable
+  - This can be overruled when necessary, f.e. if all looks good and the changes are needed for further development and the author is AFK.
```
Comment (barrymcgee) [inline workflow/github.md]: f.e. ? Do you mean e.g. ?

**Comment by barrymcgee** [comment]
Adding `do not merge` as I'm conscious this was discussed in @nottrobin's absence and previously he was the strongest proponent for the opposite approach. 

Let's not merge until he has had a chance to review this change.

File: workflow/github.md (hunk 1 above)
Comment (nottrobin) [reply workflow/github.md]: I'm cool with this change. But I agree you should change it to either "e.g.", or "for example"

**Comment by nottrobin** [comment]
:+1: once "f.e." is changed to something else

File: workflow/github.md (hunk 1 above)
Comment (b-m-f) [reply workflow/github.md]: updated :)

**Comment by nottrobin** [comment]
:+1: from me

**Comment by bartaz** [comment]
👍

**Comment by steverydz** [comment]
:+1:

**Pull request merged**